native/src/widget/pane_grid/content.rs

```diff
@@ -258,7 +258,7 @@ where
                 None => self.body.overlay(children.next()?, renderer),
             }
         } else {
-            self.body.overlay(layout, renderer)
+            self.body.overlay(layout.children().last()?, renderer)
```

Comment:
This seems incorrect, as we are returning the `body` overlay directly in `layout` when no title bar is present:

https://github.com/iced-rs/iced/blob/a536f5ee0e3b17d4962526038099197ed6ccdad1/native/src/widget/pane_grid/content.rs#L153-L155